src/web/mjs/connectors/KomikTap.mjs

```diff
@@ -8,9 +8,21 @@ export default class KomikTap extends WordPressMangastream {
         super.label = 'KomikTap';
         this.tags = [ 'manga', 'webtoon', 'hentai', 'indonesian' ];
         this.url = 'https://komiktap.in';
-        this.path = '/manga/?list';
+        this.path = '/manga/list-mode/';
 
-        this.queryMangas = 'div#content div.soralist ul li a.series';
-        this.queryChapters = 'div.bxcl ul li span.lchx a';
+        this.queryChapters = 'div#chapterlist ul li div.eph-num a';
+        this.queryChaptersTitle = 'span.chapternum';
+        this.queryPages = 'div#content div.wrapper script';
+
+    }
+
+    // NOTE: This web use costum lazyload methid
+    async _getPages(chapter) {
+        let uri = new URL(chapter.id, this.url);
+        let request = new Request(uri, this.requestOptions);
+        let nodelist = await this.fetchDOM(request, this.queryPages);
+        // NOTE: faster then find method
+        let data = JSON.parse(nodelist[nodelist.length-2].innerText.slice(14,-2))['sources'][0]['images'];
```

Comment:
May use `fetchRegex(...)`, seems a little bit of unreliable to access the script by index and strip of a fixed number of characters from the beginning/ending